scripts/filter-known-issues.py

```diff
@@ -59,13 +59,13 @@ def config_import_file(filename):
             # finditer() finds all the blocks and selects
             # the bits of mmapped-file that comprises
             # each--we compile it into a regex and append.
-            for m in re.finditer(config_regex, mm, re.MULTILINE):
+            for m in re.finditer(b"config_regex", mm, re.MULTILINE):
```

Comment:
I'm going to guess that multiline expression matching broke with this change.  @inakypg ?